Assets/Scripts/Cheat/Main/TowerController.cs

```diff
@@ -0,0 +1,125 @@
+﻿/*
+ * Author(s): Isaiah Mann
+ * Description: Controls all the towers in the game
+ */
+
+using UnityEngine;
+using SimpleJSON;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.EventSystems;
+
+public class TowerController : UnitController<ITower, Tower, TowerList>, ITowerController {
+	public const string TOWER_TAG = "Tower";
+	const float DRAG_HEIGHT_OFFSET = 1.25f;
+
+	public GameObject CoreOrbPrefab;
+	public GameObject CoreOrbInstance;
+	GameObject potentialPurchaseTower = null;
+	MapTileBehaviour previousHighlightedMapTile = null;
+	HashSet<TowerBehaviour> activeTowers = new HashSet<TowerBehaviour>();
+	public static List<TowerBehaviour> healTowers = new List<TowerBehaviour> (); //9/29/16
```

Comment:
I'm not sure if we want to add a second list of towers. Let's just use the HashSet since they effectively accomplish the same thing.

Additionally, preferably these vars are not static. Instead, the pattern for static access should be this:

1. Create instance variable in TowerController class
2. Create public getter method in TowerController class
3. Create public getter method in WorldController class that returns public getter method in step #2
4. This can be called statically using the singleton pattern we have setup: WorldController.Instance.GetterMethodName()